magda-csw-connector/aspect-templates/organization-details.js

```diff
@@ -1,8 +1,31 @@
 const name = transformer.getNameFromJsonOrganization(organization);
+const jsonpath = libraries.jsonpath;
+let phone = jsonpath.value(organization,"$.contactInfo[*].CI_Contact[*].phone[*].CI_Telephone[*].voice[0].CharacterString[0]._");
```

Comment:
can these be `const`s? :)

Reply:
They really should be const. Will add a new commit. 
Thanks for this 👍